src/executor.h

```
@@ -39,10 +39,13 @@ class Executor {
                  const Script* script,
                  Delegate* delegate,
                  const ShaderMap& map,
-                 ExecutionType executionType);
+                 ExecutionType executionType,
+                 bool disable_spirv_validation = false);
```

I don't like default args, I find they hide when parameters are no longer used.

---

Yes, makes sense. Removed.